auth-foundation/src/androidTest/java/com/okta/authfoundation/credential/SharedPreferencesTokenStorageTest.kt

```diff
@@ -49,6 +52,7 @@ class SharedPreferencesTokenStorageTest {
             dispatcher = Dispatchers.Unconfined,
             eventCoordinator = eventCoordinator,
             context = ApplicationProvider.getApplicationContext(),
+            keyGenParameterSpec = MasterKeys.AES256_GCM_SPEC,
```

Comment:
MasterKeys is deprecated as stated in https://developer.android.com/reference/androidx/security/crypto/MasterKeys. But, I can't seem to import MasterKey.Builder in this file as recommended by docs. So, not sure how to use the recommended API here. Maybe a library needs updating 🤷

Reply:
The latest release `1.1.0-alpha03`, MasterKeys is deprecated. I prototyped this to migrate to the new MasterKey, once 1.1 is stable. And it'll work! Using the latest stable for now.